app/Actions/CreateDatabase.php

```diff
@@ -38,24 +49,13 @@ public function __invoke()
                 ->create($db_name);
 
             if (! $databaseCreated) {
-                return $this->consoleWriter->warn($this->failureToCreateError($db_name));
+                $this->consoleWriter->warn($this->failureToCreateError($db_name));
             }
         } catch (PDOException $e) {
             $this->consoleWriter->warn($e->getMessage());
-            return $this->consoleWriter->warn($this->failureToCreateError($db_name));
+            $this->consoleWriter->warn($this->failureToCreateError($db_name));
         }
 
```

Comment:
Don't we need these `return` lines so we don't show `Created a new database` after a failure?